pkg/manifest/manifest_test.go

```diff
@@ -155,6 +161,100 @@ data:
 
 }
 
+func TestParseManifestsDuplicates(t *testing.T) {
+	tests := []struct {
+		name             string
+		raw              string
+		wantNumDupErrors int
+		wantErr          bool
```

Comment:
a boolean leaves a lot of room for "actual error message doesn't look like the one I expected when writing the tests".  I prefer an `expectedError string` with handling that is either a direct string match ([example][1]) or a regexp match ([example][2]).

[1]: https://github.com/openshift/cluster-version-operator/blob/9c0b0b446ed201999c7cbae8b3d6ade24f531353/pkg/cvo/cvo_test.go#L3359-L3367
[2]: https://github.com/openshift/installer/blob/c9171cfa4017771d696bbad8b70e6c2edaa244f4/pkg/types/validation/installconfig_test.go#L1274-L1277

Reply:
This is changed and tightened up.